src/main/java/proto/Pose.proto

```diff
@@ -0,0 +1,22 @@
+/*
+Instead of encoding all data in one proto, we will instead preface the proto
+with a header field containing the number of total bytes in the proto. This will
+be a configurable value that will be computed where the biggest possible Data
+division below the maximum threshold is used. This will allow for easy chunking
+when reading in messages etc.
```

Comment:
The file comment should describe the proto, what it is used for.
I'm confused why you need a header field containing the number of total bytes in the proto?